unit-tests/shared/src/test/scala/org/scalanative/testsuite/javalib/nio/channels/DatagramChannelTest.scala

```diff
@@ -0,0 +1,139 @@
+package org.scalanative.testsuite.javalib.nio.channels
+
+import org.junit.Test
+import org.junit.Assert._
+import org.junit.Assume._
```

Comment:
I think there are some unused imports here, probably from 
as standard development bootstrap template.
If my tired eyes have not missed anything, Assume, BeforeClass,
Ignore and Platform are all unused. 

It is a mark of skill, good design, & good implementation that 
there are no Platform specific paths.